tests/test_rooms/test_query_rooms.py

```diff
@@ -0,0 +1,16 @@
+from tests.base import BaseTestCase
+
+from fixtures.room.room_fixtures import (
+    rooms_query,
+    query_rooms_response,
+    room_query_by_id,
+    room_query_by_id_response
+)
+class QueryRooms(BaseTestCase):
+    def test_query_rooms(self):
```

Comment:
You might need to add more test cases to raise test coverage to atleast 70 %

Reply:
Thanks have noted this and done accordingly.